jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -422,59 +426,63 @@
             ./hack/jenkins/build.sh
             if [[ "${{ghprbTargetBranch:-}}" == "release-1.0" || "${{ghprbTargetBranch:-}}" == "release-1.1" ]]; then
               ./hack/jenkins/e2e.sh
-            else
-              # Nothing should want Jenkins $HOME
-              export HOME=${{WORKSPACE}}
-
-              export KUBERNETES_PROVIDER="gce"
-              export E2E_MIN_STARTUP_PODS="1"
-              export KUBE_GCE_ZONE="us-central1-f"
-              export FAIL_ON_GCP_RESOURCE_LEAK="true"
-
-              # Experimental flake detection-- harmlessly does nothing until
-              # ginkgo changes get made, but allows for testing them in a PR.
-              export GINKGO_TOLERATE_FLAKES="y"
-
-              export E2E_NAME="e2e-gce-${{NODE_NAME}}-${{EXECUTOR_NUMBER}}"
-              export GINKGO_PARALLEL="y"
-              # This list should match the list in kubernetes-e2e-gce.
-              export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
-              export FAIL_ON_GCP_RESOURCE_LEAK="false"
-              export PROJECT="k8s-jkns-pr-gce"
-              # NUM_NODES should match kubernetes-e2e-gce.
-              export NUM_NODES="3"
-
-              # Assume we're upping, testing, and downing a cluster
-              export E2E_UP="true"
-              export E2E_TEST="true"
-              export E2E_DOWN="true"
-
-              # Skip gcloud update checking
-              export CLOUDSDK_COMPONENT_MANAGER_DISABLE_UPDATE_CHECK=true
-
-              # GCE variables
-              export INSTANCE_PREFIX=${{E2E_NAME}}
-              export KUBE_GCE_NETWORK=${{E2E_NAME}}
-              export KUBE_GCE_INSTANCE_PREFIX=${{E2E_NAME}}
-
-              # Get golang into our PATH so we can run e2e.go
-              export PATH=${{PATH}}:/usr/local/go/bin
+              exit
+            fi
+            # Nothing should want Jenkins $HOME
```

Comment:
GitHub is doing a poor job of illustrating this diff, but I just made the release-1.0/release-1.1 case exit so I could remove the "else" and all of its indentation.